docs/en/latest/practices/the-hard-way.md

```diff
@@ -443,11 +447,13 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: apisix-ingress-controller
+  namespace: apisix
```

Comment:
Do we need to hardcode the namespace? It is already configured in the command line.

Reply:
This doc tells us to install apisix-ingress-controller manually but not by helm. And all the configurations are all deployed in `apisix` namespace, deployment/serviceAccount/clusterRole, etc. So I think we should specify this parameter explicitly.

Reply:
I agree, we can keep them consistent